### PR TITLE
Move streambench log for spark to report dir

### PR DESCRIPTION
**Pull request by qiansl127**

### CODE REVIEW

**Comment by qiansl127** [comment]
@carsonwang Please review this!

File: src/streambench/sparkbench/src/main/scala/com/intel/hibench/streambench/spark/RunBench.scala [hunk 1]
```diff
@@ -49,6 +51,8 @@ object RunBench {
     val brokerList = if (directMode) conf.getPropertiy("hibench.streamingbench.brokerList") else ""
     val totalParallel = conf.getPropertiy("hibench.yarn.executor.num").toInt * conf.getPropertiy("hibench.yarn.executor.cores").toInt
 
+    this.reportDir = conf.getPropertiy("hibench.report.dir")
```
Comment (carsonwang) [inline src/streambench/sparkbench/src/main/scala/com/intel/hibench/streambench/spark/RunBench.scala]: Can you fix the typo here? `getPropertiy` -> `getProperty`. Maybe another PR.

Comment (qiansl127) [inline src/streambench/sparkbench/src/main/scala/com/intel/hibench/streambench/spark/RunBench.scala]: Done.

**Comment by carsonwang** [comment]
Ok, merging it now.

**Pull request merged**